lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -825,3 +826,25 @@ sub sdaf_ansible_verbosity_level {
     return '-vvvv';    # Default set to "-vvvv"
 }
 
+=head2 ansible_hanasr_show_status
+
+    ansible_hanasr_show_status(sdaf_config_root_dir=>'/some/path' [, sap_sid=>'CAT']);
+
+Display simple command outputs from all DB hosts using B<ansible> command.
+
+B<sdaf_config_root_dir> SDAF Config directory containing SUT ssh keys
+
+B<sap_sid>: SAP system ID. Default 'SAP_SID'
+
+=cut
+
+sub ansible_hanasr_show_status {
+    my (%args) = @_;
+    $args{sap_sid} //= get_required_var('SAP_SID');
+    croak 'Missing mandatory argument "sdaf_config_root_dir".' unless $args{sdaf_config_root_dir};
+
+    my @cmd = ('ansible', 'QES_DB', "--private-key=$args{sdaf_config_root_dir}/sshkey", "--inventory=$args{sap_sid}_hosts.yaml", '--module-name=shell');
```

Comment:
Is there a reason you are not using the concatenation operator?

Reply:
I prefer using  `join(' ', @cmd)` so it handles spaces between arguments for me.